components/cam/src/physics/cam/micro_p3_utils.F90

```diff
@@ -1,7 +1,26 @@
+
+#ifdef SCREAM_CONFIG_IS_CMAKE
```

Comment:
I'm surprised you needed the bfb math layer for this file. We should probably move it to it's own header and #include it instead of duplicating it.

Reply:
That's correct the math layer is not needed here, I have thus removed it from this file.